test/spec/vis/vis.spec.js

```diff
@@ -303,12 +303,44 @@ describe('vis/vis', function () {
         expect(this.vis._windshaftMap.createInstance).toHaveBeenCalledWith({
           sourceId: 'sourceId',
           forceFetch: 'forceFetch',
-          success: 'success'
+          success: 'success',
+          includeFilters: true
         });
       });
     });
   });
 
+  describe('.init', function () {
```

Comment:
There's no `.init` method anymore, so these tests should be part of `.createInstance`